video_store.py

```diff
@@ -0,0 +1,139 @@
+import requests
```

Comment:
Ideally, file names should be the same name as the class within them. So in this case, this file should be named 'employee.py' but for now, it's okay. Just food for thought in the future :)